x-pack/plugins/security/server/routes/authentication/common.ts

```diff
@@ -57,11 +59,27 @@ export function defineCommonRoutes({
           excludeFromOAS: true,
           authRequired: false,
           tags: [ROUTE_TAG_CAN_REDIRECT, ROUTE_TAG_AUTH_FLOW],
+          ...(isDeprecated && {
+            deprecated: {
+              documentationUrl: 'https://elastic.co', // ToDo: Update with docLink to upgrade note
```

Comment:
With the PR known and the release note added, I think we can go ahead and populate this following [the template](https://github.com/elastic/kibana/blob/9d38922401d0bbd0d95d750f68fec77ca22758fb/docs/upgrade-notes.asciidoc?plain=1#L5)'s instructions about doc links. Ditto for other instances!

Reply:
@jloleysens Done in 97153f16fe1fd349408dc4073280dd6625c543d0

Could you have a quick look?